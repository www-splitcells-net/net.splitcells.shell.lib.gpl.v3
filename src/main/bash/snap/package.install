#!/usr/bin/env bash
# SPDX-License-Identifier: GPL-3.0-only
# SPDX-FileCopyrightText: Contributors To The `net.splitcells.*` Projects
installationLog=$(sudo snap install $1) # You can search for packages at "https://snapcraft.io/store".
exitCode=$?
echo.error $installationLog
exit $exitCode
