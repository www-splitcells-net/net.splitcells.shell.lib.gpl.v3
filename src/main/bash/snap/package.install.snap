#!/usr/bin/env bash
# SPDX-License-Identifier: GPL-3.0-only
# SPDX-FileCopyrightText: Contributors To The `net.splitcells.*` Projects
# Snap client software seems to be licensed under GPL 3: https://en.wikipedia.org/wiki/Snappy_(package_manager)
package.install snapd

if [ -x "$(command -v setenforce)" ]; then # Check if SElinux is present.
	sudo.at.current.env 'link.symbolic.from /snap to /var/lib/snapd/snap' # Enable support for --classic.
fi

sleep 3

sudo systemctl enable snapd
sudo systemctl start snapd

if systemctl is-active --quiet snapd; then
	echo.error "Snapd is not running." | systemd-cat -p emerg
fi